proxysql-status

```diff
@@ -16,6 +16,8 @@ Usage example:
     --stats                 : display stats tables
     --table=<table_name>    : display only tables that contain the table name
                               (note: this is a case-sensitive match)
+    --with-stats-reset      : display _reset tables, by default _reset tables
+                              will not query.
```

Comment:
"will not query"  should be "will not be queried"

Reply:
done